tests/unit/parts/test_lifecycle.py

```diff
@@ -1571,9 +1575,6 @@ def test_patch_elf(snapcraft_yaml, mocker, new_dir):
             patchelf_args=[
                 "--set-interpreter",
                 "/snap/core22/current/lib64/ld-linux-x86-64.so.2",
-                "--force-rpath",
```

Comment:
These lines can be removed because [this if statement](https://github.com/snapcore/snapcraft/blob/13719604c44d8761df3154cd964f29d290e8a563/snapcraft/elf/_patcher.py#L67) will always be false.

Reply:
Shouldn't we unit-test that codepath as well?

Reply:
That codepath in `snapcraft/elf/_patcher.py.Patcher.patch()` is already unit tested by `tests/unit/elf/test_patcher.py`.